advice/jkraft-advice.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <link rel="stylesheet" type="text/css" href="../css/style.css">
```

Comment:
This is the link that needs to be changed. :)

Reply:
Hey @msjkraft! You mention earlier in this conversation that you had changed the link to your CSS file - but that edit never made it to your pull request. Can you double-check the result of `git status` to make sure that you don't have any changes that still needs to be added and/or committed? Once the result of git status is clean, please try pushing up your branch again with `git push origin jkpractice`. 

Keep us posted and let us know if you have more questions!

Cheers,
Ann